playbook-website/config/webpack/environment.js

```diff
@@ -12,6 +12,20 @@ environment.loaders.insert('javascript', {
   },
   include: path.resolve(__dirname, '../../app'),
 })
+
+// Allow ESM modules
+environment.config.merge({
+  module: {
+    rules: [
+      {
+        test: /\.mjs$/,
+        include: /node_modules/,
+        type: 'javascript/auto',
+      },
+    ],
+  },
+})
+
```

Comment:
This gets `webpacker` past the `Can't import the named export...` error. By splitting this config out from the `javascript` one, we bypass `babel-loader` which isn't necessary as the source is already transpiled.